java/connector-node/risingwave-connector-service/src/main/resources/mysql.properties

```diff
@@ -4,25 +4,19 @@ offset.storage=com.risingwave.connector.cdc.debezium.internal.ConfigurableOffset
 database.history=io.debezium.relational.history.MemoryDatabaseHistory
 # default snapshot mode to initial
 snapshot.mode=${debezium.snapshot.mode:-initial}
-
 database.hostname=${hostname}
 database.port=${port}
 database.user=${username}
 database.password=${password}
-
 database.include.list=${database.name}
 table.include.list=${database.name}.${table.name}
-
 # default to disable schema change events
 include.schema.changes=${debezium.include.schema.changes:-false}
 database.server.id=${server.id}
-
 # set connector timezone to UTC(+00:00)
 database.connectionTimeZone=+00:00
-
 # default heartbeat interval 60 seconds
 heartbeat.interval.ms=${debezium.heartbeat.interval.ms:-60000}
 heartbeat.topics.prefix=${debezium.heartbeat.topics.prefix:-RW_CDC_HeartBeat_}
-name=${hostname}:${port}:${database.name}.${table.name}
-
+name=${hostname}:${port}:${database.name}.${table.name:-rw}
```

Comment:
Why do we have a default value for the table name? 👀

Reply:
I guess it would be better to use a more specified default value here, or leave some comments.